cypress/e2e/bulk-edit/csv/bulk-edit-csv-drag-and-drop-is-enabled-after-file-is-uploaded.cy.js

```diff
@@ -0,0 +1,63 @@
+import getRandomPostfix from '../../../support/utils/stringTools';
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import FileManager from '../../../support/utils/fileManager';
+
+describe('bulk-edit', () => {
+  const testData = {};
+  const userUUIDsFileName = `userUUIDs_${getRandomPostfix()}.csv`;
+  const invalidUserUUID = getRandomPostfix();
+
+  before('Create test data', () => {
+    cy.getAdminToken().then(() => {
+      // create all test objects
+    });
+
+    cy.createTempUser([
+      Permissions.bulkEditCsvView.gui,
+      Permissions.bulkEditCsvEdit.gui,
+      Permissions.uiUsersView.gui,
+    ]).then((userProperties) => {
+      testData.user = userProperties;
+
+      cy.login(testData.user.username, testData.user.password, {
+        path: TopMenu.bulkEditPath,
+        waiter: BulkEditSearchPane.waitLoading,
+      });
+      FileManager.createFile(
```

Comment:
No need create file every run. Just put file from TC to fixtures folder
In case of  attached file rom test case  is not displayed in test.io 
[user_uuid_val.csv](https://github.com/folio-org/stripes-testing/files/13174685/user_uuid_val.csv)

Reply:
Thank you. Fixed

Reply:
@IlyaLyskavets @manowarqqq please revert back to the creation of file in preconditions

Reply:
reverted to previous implementation